components/customcomponents/CustomAddToCart.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <div class="sf-add-to-cart">
+    <slot name="quantity-select-input" v-bind="{ qty }">
+      <div class="column">
+        <SfQuantitySelector
+          :qty="qty"
+          :max="quantityleft"
+          aria-label="Quantity"
+          :disabled="disabled"
+          class="sf-add-to-cart__select-quantity"
+          @input="$emit('input', $event)"
+        />
+      </div>
+    </slot>
+    <slot name="add-to-cart-btn">
+      <!--@slot Custom content that will replace default Add to cart button design.-->
+      <div class="column">
+        <SfButton class="sf-add-to-cart__button" :disabled="disabled" @click="addToCart">
+          Add to cart
+        </SfButton>
+      </div>
+    </slot>
+    <slot name="item-left-span">
+      <div class="column">
+        <span class="quantity-left">{{ quantityleft }} item(s) left</span>
+      </div>
+    </slot>
+    <slot name="add-to-wishlist">
+      <div class="column">
+        <SfButton class="sf-add-to-wishlist__button" :disabled="disabled" @click="addToWishList">
+          Add to Wishlist
+        </SfButton>
+      </div>
+    </slot>
+  </div>
+</template>
+<script>
+import { SfButton, SfQuantitySelector } from "@storefront-ui/vue"
+export default {
+  name: "CustomAddToCart",
+  components: {
+    SfButton,
+    SfQuantitySelector,
+  },
+  model: {
+    prop: "qty",
+  },
+  props: {
+    /**
+     * Boolean to indicate whether product
+     * can be added to cart
+     */
+    disabled: {
+      type: Boolean,
+      default: false,
+    },
+    /**
+     * Selected quantity
+     */
+    qty: {
+      type: [Number, String],
```

Comment:
Do we need String here?

Reply:
Existing  SfAddToCart component have both the types so kept as it is